src/main/java/me/dinowernli/grpc/polyglot/config/CommandLineArgs.java

```diff
@@ -153,6 +163,25 @@ private CommandLineArgs() {
     return maybePath(tlsCaCertPath);
   }
 
+  public Optional<Multimap<String, String>> metadata() {
```

Comment:
Please make this one of the guava immutable multimaps (so that all returned fields here stay immutable)